src/MarginTrading.Brokers/MarginTrading.OrderbookBestPricesBroker/Program.cs

```diff
@@ -0,0 +1,12 @@
+﻿using MarginTrading.BrokerBase;
+
+namespace MarginTrading.OrderbookBestPricesBroker
+{
+    public class Program : WebAppProgramBase<Startup>
+    {
+        public static void Main(string[] args)
+        {
+            RunOnPort(5016);
```

Comment:
Port does not match kubernetes-dev settings - there it is 5014.

Reply:
Fixed